adazza-service.gradle

```diff
@@ -19,6 +19,7 @@ subprojects {
 }
 
 apply from: "https://raw.githubusercontent.com/adazza/adazza-gradle/${gradleBranch}/adazza-core.gradle"
+apply from: "https://raw.githubusercontent.com/adazza/adazza-gradle/${gradleBranch}/adazza-scalariform.gradle"
```

Comment:
This would work better if you applied it in -scala and hooked it up to scala compile.